discovery-service/src/main/java/com/ca/mfaas/discovery/staticdef/ServiceDefinitionProcessor.java

```diff
@@ -178,7 +178,7 @@ private void setInstanceAttributes(InstanceInfo.Builder builder,Service service,
             .setMetadata(createMetadata(service, url, tile));
 
         if (service.getHomePageRelativeUrl() != null) {
-            builder.setHomePageUrl(null, instanceBaseUrl + service.getHomePageRelativeUrl());
+            builder.setHomePageUrl(null, service.getHomePageRelativeUrl());
```

Comment:
This is changing the meaning of the `homePageRelativeUrl` property that is documented in https://zowe.github.io/docs-site/latest/extend/extend-apiml/api-mediation-onboard-an-existing-rest-api-service-without-code-changes.html#configuration-parameters and used by Zowe components

* **homePageRelativeUrl**

    Specifies the relative path to the homepage of your service. The path should start with `/`.
    If your service has no homepage, omit this parameter.

    **Examples:**
    * `homePageRelativeUrl: /` The service has homepage with URL `${baseUrl}/`
    * `homePageRelativeUrl: /ui/` The service has homepage with URL `${baseUrl}/ui/`
    * `homePageRelativeUrl: ` The service has homepage with URL `${baseUrl}`

Reply:
Hi Petr, thank you for the clarification! I'll try to address all your requests.